internal/impl/babysitter.go

```diff
@@ -176,8 +183,18 @@ func (b *babysitter) watchPods(ctx context.Context, component string) error {
 		return fmt.Errorf("unable to determine group name for component %s", component)
 	}
 	name := deploymentName(b.app.Name, rs, b.cfg.DeploymentId)
+	//timeOut := int64(60)
+	// , TimeoutSeconds: &timeOut
```

Comment:
This is the timeout. The way it usually is done is like:
```go
	timeOut := int64(10)
	watchFunc := func(options metav1.ListOptions) (apiWatch.Interface, error) {
		// Return watcher with 10m timeout
		return clientset.CoreV1().Namespaces().Watch(context.Background(), metav1.ListOptions{TimeoutSeconds: &timeOut})
	}
```

Let me know if you guys think it should be done this way or not... for now, I will be testing without it.

Reply:
I don't think we should add any timeout. Let's rely on the default value for the timeout which supposed to be 30 minutes.